src/components/dar_collection_table/DarCollectionTableCellData.js

```diff
@@ -1,13 +1,14 @@
-import {isEmpty, isNil} from "lodash/fp";
+import {includes, isEmpty, isNil, toLower} from "lodash/fp";
 import {formatDate} from "../../libs/utils";
 import {h} from "react-hyperscript-helpers";
 import {styles} from "./DarCollectionTable";
 import AdminActions from "./AdminActions";
 import ChairActions from "./ChairActions";
 import MemberActions from './MemberActions';
 import ResearcherActions from './ResearcherActions';
+import {Link} from "react-router-dom";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9480935)
```suggestion
import {Link} from 'react-router-dom';
```